README.md

```
@@ -20,8 +35,11 @@ npm install ngx-stories
 ```
 
 ```ts
+### Import the component:
 import { NgxStoriesComponent, StoryGroup } from 'ngx-stories';
-
+```
+## Usage
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)